.travis.yml

```diff
@@ -16,9 +16,19 @@ addons:
 install:
   - ./mvnw -T 1.5C clean install -DskipTests=true -Dmaven.javadoc.skip=true -B -V -q
 
+before_install:
+  - if [ "$TRAVIS_SECURE_ENV_VARS" == "true" ]; then
+    openssl aes-256-cbc -K $encrypted_5bb5dbfd81fd_key -iv $encrypted_5bb5dbfd81fd_iv -in travis.tar.gz.enc -out travis.tar.gz -d
```

Comment:
semicolon here too?

Reply:
Done.

Reply:
Don't see it. I think you modified the wrong line.

Reply:
Should be good now. 

Now the build/test runs but getting error related to sonar plugin: https://travis-ci.org/GoogleCloudPlatform/google-cloud-spanner-hibernate/builds/505409847?utm_source=github_status&utm_medium=notification

Any ideas? I haven't yet added `&& [ "$TRAVIS_PULL_REQUEST" == "false" ]` to the before_install yet; might be related to that?

Reply:
Yeah, the sonar build on master has to succeed first and I'm not sure why it has been failing. The setup seems the same as for S-C-G. Let's just merge for now.